allensdk/brain_observatory/behavior/session_apis/data_transforms/behavior_ophys_data_xforms.py

```diff
@@ -127,25 +106,49 @@ def get_ophys_timestamps(self):
         return ophys_timestamps
 
     @memoize
-    def get_experiment_container_id(self):
-        query = '''
-                SELECT visual_behavior_experiment_container_id
-                FROM ophys_experiments_visual_behavior_experiment_containers
-                WHERE ophys_experiment_id= {};
-                '''.format(self.get_ophys_experiment_id())
-        return self.lims_db.fetchone(query, strict=False)
+    def get_sync_data(self):
```

Comment:
Again I'm not quite sure about the separation here -- most of the transforms rely on this on-disk file.